cmake/FindWPE.cmake

```diff
@@ -29,19 +29,29 @@
 # ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 find_package(PkgConfig)
-pkg_check_modules(PC_WPE QUIET wpe-1.0)
+pkg_check_modules(WPE IMPORTED_TARGET wpe-1.0)
 
-find_path(WPE_INCLUDE_DIRS
+find_path(WPE_INCLUDE_DIR
     NAMES wpe/wpe.h
-    HINTS ${PC_WPE_INCLUDEDIR} ${PC_WPE_INCLUDE_DIRS}
+    HINTS ${WPE_INCLUDEDIR} ${WPE_INCLUDE_DIRS}
 )
-
-find_library(WPE_LIBRARIES
+find_library(WPE_LIBRARY
     NAMES wpe-1.0
-    HINTS ${PC_WPE_LIBDIR} ${PC_WPE_LIBRARY_DIRS}
+    HINTS ${WPE_LIBDIR} ${WPE_LIBRARY_DIRS}
 )
+mark_as_advanced(WPE_INCLUDE_DIR WPE_LIBRARY)
 
-mark_as_advanced(WPE_INCLUDE_DIRS WPE_LIBRARIES)
+# If pkg-config has not found the module but find_path+find_library have
+# figured out where the header and library are, create the PkgConfig::WPE
+# imported target anyway with the found paths.
+#
+if (WPE_LIBRARY AND NOT TARGET PkgConfig::WPE)
```

Comment:
Doesn't this seem like a lot of extra work for the insane case that libwpe is installed but the pkg-config is mysteriously broken? I think if pkg-config doesn't return a result then we should just trust that it's not found?

Reply:
We need this as a fallback for systems like Windows where getting `pkg-config` to work is a hit-and-miss task. As you pointed out on IRC in theory [pkgconf](http://pkgconf.org/) works on Windows, but I don't know how widespread it is (or how well known it is that it works) so it seems better to keep the fallback and avoid breaking people's build setups around.

I would still want to revisit this topic in the future, try things, and see how far we could get with `pkgconf`, but I would say that such thing would be out of the scope of this PR.